comms_webrtc.orogen

```diff
@@ -2,54 +2,25 @@ name "comms_webrtc"
 # Optionally declare the version number
 # version "0.1"
 
-# If new data types need to be defined, they have to be put in a separate C++
-# header, and this header will be loaded here
 import_types_from "comms_webrtcTypes.hpp"
-# Finally, it is pretty common that headers are directly loaded from an external
-# library. In this case, the library must be first used (the name is the
-# library's pkg-config name) and then the header can be used. Following Rock
-# conventions, a common use-case would be:
-#
-# using_library "comms_webrtc"
-# import_types_from "comms_webrtc/CustomType.hpp"
+import_types_from "iodrivers_base"
+import_types_from "base"
 
-# If this project uses data types that are defined in other oroGen projects,
-# these projects should be imported there as well.
-import_types_from "std"
-# import_types_from "base"
+using_library "libdatachannel"
+using_library "jsoncpp", typekit: false
 
-# Declare a new task context (i.e., a component)
-#
-# The corresponding C++ class can be edited in tasks/Task.hpp and
-# tasks/Task.cpp, and will be put in the comms_webrtc namespace.
-task_context "Task" do
-    # This is the default from now on, and should not be removed. Rock will
-    # transition to a setup where all components use a configuration step.
+task_context "WebRTCTask" do
```

Comment:
Just "Task" is a good name in a package that will probably have only one.

The webrtc part is already in the project name.